pmlc/dialect/pxa/transforms/fusion.cc

```diff
@@ -309,10 +338,14 @@ struct FusionInfo {
     }
     // For each raw & waw, consider the plan
     for (auto &raw : readAfterWrites) {
-      considerPlan(raw.first, raw.second);
+      auto write = cast<AffineWriteOpInterface>(raw.first);
+      auto read = cast<AffineReadOpInterface>(raw.second);
```

Comment:
PXA doesn't actually implement these interfaces. We should probably create our own PXA based interfaces, and we've talked about doing so, but it hasn't been implemented yet.

Reply:
I saw that Affine interface was used for load and vector_load ops, so added the same to reduce. Let me separate these two based on the Affine ones.